install_config/router/f5_router.adoc

```diff
@@ -178,14 +178,23 @@ partition.
 See the
 link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos_management_guide_10_0_0/tmos_partitions.html[*F5 BIG-IP®* documentation] for more information about administrative partitions.
 
-Use the `--external-host-partition-path` flag when
-xref:deploying-the-f5-router[deploying the F5 router] to specify a partition
-path:
+To configure your {product-title} for partition paths:
 
+. Ensure F5 is configured to be able to switch to the */Common* and */Custom* paths. See
+the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-implementations-12-0-0/9.html[*F5
+BIG-IP®* documentation] for more information.
+
```

Comment:
this step is not always needed, it just "Ensure the related openshift configuration on F5 is clear up". Most clear up steps are straightforward, the link just has the information about how to "delete static FDB". 
Or we needn't to provide the external link just ask user to ensure F5 in its initial configuration.